### DISCUSSION TITLE
My Wallet can not open

### DISCUSSION THREAD

**Discussion by SAULAME** [iotaledger/firefly#7705] [Help & Support]
I have a problem I just made an actualization of my Firefly, and  once the actualization finished and i want to open my wallet  the ssytem show me this message :Mon, 13 Nov 2023 00:04:42 GMT
wallet: `local time 1699833882 doesn't match the time of the latest milestone timestamp: 1699830191`

And the wallet can not open   what i must do?

**Comment by JannemanDev** [comment]
Your clock is more than an hour off real time! Check internet connection and resync your clock.
